### PR TITLE
Minor UI improvements in HTML layout, alignment, security fixes as per Chrome's Lighthouse recommendations

### PR DESCRIPTION
**Pull request by vkirienko**

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,16 +1,17 @@
 ﻿<!DOCTYPE HTML>
 <html>
 
-  <head>
-    <meta charset="utf-8"/>
+<head>
+    <meta charset="utf-8" />
+    <meta name="description" content="Improved dump1090-fa web interface">
```
Comment (llacb47) [inline html/index.html:6]: I don't think this is a good description, just my opinion. How about: `Tar1090 - Live flight tracking map brought to you by ADS-B Exchange`? Or something a little more SEO-ey? :)

File: html/index.html [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui" />
 
-    <script src="jquery/jquery-3.4.1.min.js"></script>
+    <script src="jquery/jquery-3.5.1.min.js"></script>
```
Comment (llacb47) [inline html/index.html:10]: These files don't exist (yet): https://github.com/wiedehopf/tar1090/tree/master/html/jquery

File: html/index.html (hunk 1 above)
Comment (vkirienko) [reply html/index.html:6]: I agree it sounds much better but just copied it from this repo readme. I think it is up to wiedehopf to change it.

**Comment by wiedehopf** [comment]
What is the package json and lock json required for?

**Comment by vkirienko** [comment]
Javascript project manifest (that this project basically is). Not required if you don't want to use it but convenient way to specify all external dependencies, automatically pull them with npm-check-updates and etc. 

https://nodejs.org/en/knowledge/getting-started/npm/what-is-the-file-package-json/
https://flaviocopes.com/package-json/

**Comment by wiedehopf** [comment]
I despise npm with a passion.
I'll keep it simpler (for me) and update the libraries manually. (at least for the moment)

I also don't have use for the gitignore file but i might add that anyhow because it doesn't hurt and doesn't need updating.

I'm not sure how bugs for example in jquery could be security relevant?

Thank you for the close button and taking a look.
I'll probably take the referrer patches and w/e that invalid html is ;)

If you want to help with some more html, you could continue this: https://github.com/wiedehopf/tar1090/commit/637042eb06f30048f92800abee3feabc574bc38f
I'd like to avoid sometimes the data being displayed with the label below ....

I should probably document this, but to test html changes on your local ADS-B receiver, you can run
`sudo bash install.sh test` 
This will use the files in the present directory for the install instead of fetching from github.

**Comment by vkirienko** [comment]
> I despise npm with a passion.
> I'll keep it simpler (for me) and update the libraries manually. (at least for the moment)

But it does not hurt to have it if you change your mind at some point.
 
> I also don't have use for the gitignore file but i might add that anyhow because it doesn't hurt and doesn't need updating.

It is useful for somebody like me who uses Visual Studio for development and not just notepad. :)
 
> I'm not sure how bugs for example in jquery could be security relevant?

In general my philosophy is to always run on latest third-party packages and pay attention to what security tools tell me even if security problem (no matter how remote) unlikely and/or I don't understand it's impact. 

> Thank you for the close button and taking a look.
> I'll probably take the referrer patches and w/e that invalid html is ;)

w/e - what it means? Invalid HTML in this case was missing <td></td> inside <tr>. One or two cases.
 
> If you want to help with some more html, you could continue this: [637042e](https://github.com/wiedehopf/tar1090/commit/637042eb06f30048f92800abee3feabc574bc38f)
> I'd like to avoid sometimes the data being displayed with the label below ....

I'll check.
 
> I should probably document this, but to test html changes on your local ADS-B receiver, you can run
> `sudo bash install.sh test`
> This will use the files in the present directory for the install instead of fetching from github.

That's nice to know even it is not applicable for me as I do development on Windows. Is there a way to point local Windows copy to existing receiver on Raspberry?

**Comment by vkirienko** [comment]
> If you want to help with some more html, you could continue this: [637042e](https://github.com/wiedehopf/tar1090/commit/637042eb06f30048f92800abee3feabc574bc38f)
> I'd like to avoid sometimes the data being displayed with the label below ....

Do you have more information on what problem is? Screenshot maybe? And what you are trying to acomplish?

**Comment by wiedehopf** [comment]
> Do you have more information on what problem is? Screenshot maybe? And what you are trying to accomplish?

So the aircraft details are just divs below each other.
Suppose you have a long country name, Kingdom of the Netherlands.
It would display:
Country:
Type code: Kingdom of the Netherlands

I fixed that by making the top box a table, making a row from every div.
Maybe you have a better solution?
It's not that important i'll just do it myself at some point ;)

w/e means whatever

In regards to pointing your local windows copy to json files ....
I suppose you could use nginx with the directive proxy_pass for the /data and /chunks directories.

And don't call vim notepad please ;)

**Comment by wiedehopf** [comment]
I've cherry picked a couple of commits / changed some stuff.
Some other stuff i didn't take.
Updates done manually, taking the library updates as a PR would be anti-security really.

I'll count this as somewhat applied and close the PR.
Please do a reset --hard in case you want to do further changes.

The rowspacer change ... makes the html more ugly to read and it wasn't done for all instances.
Newline changes .... meh some of those were on purpose for some resemblence of readability.

**Pull request closed without merging**